openslides/core/static/js/core/projector.js

```diff
@@ -171,7 +171,7 @@ angular.module('OpenSlidesApp.core.projector', ['OpenSlidesApp.core'])
         var STEPS = 5;
         $scope.scroll = 0;
         var setScroll = function (scroll) {
-            scroll = 80 * scroll;
+            scroll = -80 * scroll;
```

Comment:
@emanuelschuetze Does you projector scrolls also in the wrong direction in current master? (Should be fixed with this)